bin/packages/build.js

```diff
@@ -62,18 +70,51 @@ function getBuildPath( file, buildFolder ) {
 }
 
 /**
- * Build a file for the required environments (node and ES5)
+ * Given a list of scss and js filepaths, divide them into sets them and rebuild.
+ *
+ * @param {Array} files list of files to rebuild
+ */
+function buildFiles( files ) {
+	// Reduce files into a unique sets of javaScript files and scss packages.
+	const buildPaths = files.reduce( ( accumulator, filePath ) => {
+		if ( isJsFile( filePath ) ) {
+			accumulator.jsFiles.add( filePath );
+		} else if ( isScssFile( filePath ) ) {
+			const pkgName = getPackageName( filePath );
+			const pkgPath = path.resolve( PACKAGES_DIR, pkgName );
```

Comment:
In another PR #8287 I have the use-case for multiple SASS entry points (multiple CSS outputs) per package. Do you have some ideas on how this would play together?

Reply:
The simplest way I can think of is to add some kind of config, potentially as a custom property to the package.json that could be picked up by the build scripts:
```
{
  scssEntryPoints: [
    'style.scss',
    'some/other/style.scss'
  ],
}
```

Either that or some kind of naming convention e.g. `styles.main.scss`. 

I think the config idea is safer.